File: clojure/clojure-vs/update-all.clj
(in-ns 'user)

(require '[clojure.java [io :refer [file]] 
                         [shell :refer [sh]]])

(let [files (-> (file ".") (.listFiles))
      projects (filter #(.isDirectory %) files)]
  (doseq [project projects]
    (println "Update" (str (.getName project) "..."))
    (println (:out (sh "git" "pull" :dir project))))
  )
(System/exit 0)

File: clojure/clojure-vs/update-all.py
#!/usr/bin/python

import os, subprocess

projects = os.walk('.').next()[1]
for project in projects:
    print 'update {0} project...'.format(project)
    os.chdir(project)
    subprocess.call(['git', 'pull'])
    print
    os.chdir('..')
    
